src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -0,0 +1,231 @@
+your web app. Your `start_url` must reside within the `scope`.
+
+{% Aside 'caution' %}
+If the user clicks a link in your app that navigates outside of the
+`scope`, the link will open and render within the existing PWA window. If
+you want the link to open in a browser tab, you must add `target="_blank"`
+to the `<a>` tag. On Android, links with `target="_blank"` will open in a
+Chrome Custom Tab.
+{% endAside %}
+
+A few other notes on `scope`:
+
+* If you don't include a `scope` in your manifest, then the default implied
+  `scope` is the directory that your web app manifest is served from.
+* The `scope` attribute can be a relative path (`../`), or any higher level
+  path (`/`) which would allow for an increase in coverage of navigations
+  in your web app.
+* The `start_url` must be in the scope.
+* The `start_url` is relative to the path defined in the `scope` attribute.
+* A `start_url` starting with `/` will always be the root of the origin.
+
+#### `theme_color` {: #theme-color }
+
+The `theme_color` sets the color of the tool bar, and may be reflected in
+the app's preview in task switchers. The `theme_color` should match the
+`meta` theme color specified in your document head.
+
+## Add the web app manifest to your pages
+
+When you have created the manifest, add a `<link>` tag to all the pages of your
+Progressive Web App. For example:
+
+```html
+<link rel="manifest" href="/manifest.json">
+```
+
+{% Aside 'gotchas' %}
+The request for the manifest is made **without** credentials (even if it's
+on the same domain), thus if the manifest requires credentials, you must
+include `crossorigin="use-credentials"` in the manifest tag.
+{% endAside %}
+
+## Test your manifest
+
+To verify your manifest is setup correctly, use the **Manifest** tab on the
+**Application** panel of Chrome DevTools.
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled"
+       src="lh-manifest.png"
+       alt="The application panel in Chrome Devtools with the manifest tab selected.">
+</figure>
+
+This tab provides a human-readable version of many of your manifest's
+properties, and makes it easy to verify that all of the images are loading
+properly.
+
+## Splash screens on mobile {: #splash-screen }
+
+When your app first launches on mobile, it can take a moment for the browser
+to spin up, and the initial content to begin rendering. Instead of showing a
+white screen that may look to the user like the app is stall, the browser
+will show a splash screen, until the first paint.
+
+Chrome will automatically creates the splash screen from the manifest
+properties, including:
+
+* `name`
+* `background_color`
+* `icons`
+
+The `background_color` should be the same color as the load page, to provide
+a smooth transition from the splash screen to your app.
+
+Chrome will choose the icon that closely matches the device resolution for the
+device. Providing a 192px and a 512px icon is sufficient for most cases, but
```

Comment:
Delete "for the device". It's redundant.

"Providing a 192px and a 512px icon..." to "Providing 192px and 512px icons..."

Reply:
Fixed